### PR TITLE
AdsEverywhere

### PR DESCRIPTION
**Pull request by Inbestigator**
See ads everywhere, so Discord finally feels like a real website.

image

### CODE REVIEW

**Comment by Sqaaakoi** [comment]
Is the code trying to be of the same quality as the idea of the plugin itself?

**Comment by Inbestigator** [comment]
> Is the code trying to be of the same quality as the idea of the plugin itself?

The code **is** of the same quality, it's impossible to do this without dom manip afaik (if you can do it with patches, feel free to lmk)

File: src/plugins/adsEverywhere/index.ts [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+                row.append(link);
+            }
+        }
+    });
+    setTimeout(addAds, 1000);
+}
+
+export default definePlugin({
+    name: "AdsEverywhere",
+    description: "See ads everywhere, so Discord feels like a real website.",
```
Comment (sadan4) [inline src/plugins/adsEverywhere/index.ts:100]:
```suggestion
    description: "See ads everywhere, so Discord feels like a real website.",
    required: true,
```

think you forgot this

Comment (Inbestigator) [reply src/plugins/adsEverywhere/index.ts:100]: Google moment

**Pull request closed without merging**